### ISSUE TITLE
sendtonameUI expiry behaviour

### ISSUE THREAD

**Issue by phelixbtc** [namecoin/namecoin-legacy#134]
To make phishing with accidentally expired and hijacked domains more difficult it should assert that the name never expired during the last 1000 blocks or so. 

(Edited: This should go somewhere else as a more general issue: Also it might be helpful to make a name non-working when it is a 1000 blocks or less before expiry so that the holder notices it easier or can be notified.)

**Comment by JeremyRand** [comment]
Enforcing this in the GUI wouldn't affect squatters; they would just use namecoind (or a patched version, if we enforced this in namecoind too).

You did see my proposal to make names stop resolving early in NMControl, right?  https://wiki.namecoin.info/index.php?title=User:Biolizard89/ExpireWarning

**Comment by phelixbtc** [comment]
Clarified the OP. It does not protect against the squatting but against the sending of money to stolen names.

**Comment by JeremyRand** [comment]
@phelixbtc I'm fine with the 1st change.  Making a name stop working when it is close to expiry should be handled by this proposal IMHO: https://wiki.namecoin.info/index.php?title=User:Biolizard89/ExpireWarning .  (But if that proposal is accepted, then I would certainly support applying it to sendtoname.)

**Comment by phelixbtc** [comment]
@JeremyRand Yeah, the 2nd change might be of more general nature. So this issue should focus on phising prevention. Edited the OP to reflect that.
